src/NUnitFramework/framework/Internal/PropertyBag.cs

```diff
@@ -1,177 +1,178 @@
-// ***********************************************************************
-// Copyright (c) 2015 Charlie Poole, Rob Prouse
-//
-// Permission is hereby granted, free of charge, to any person obtaining
-// a copy of this software and associated documentation files (the
-// "Software"), to deal in the Software without restriction, including
-// without limitation the rights to use, copy, modify, merge, publish,
-// distribute, sublicense, and/or sell copies of the Software, and to
-// permit persons to whom the Software is furnished to do so, subject to
-// the following conditions:
-// 
-// The above copyright notice and this permission notice shall be
-// included in all copies or substantial portions of the Software.
-// 
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-// ***********************************************************************
-
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using NUnit.Framework.Interfaces;
-
-namespace NUnit.Framework.Internal
-{
-    /// <summary>
-    /// A PropertyBag represents a collection of name value pairs
-    /// that allows duplicate entries with the same key. Methods
-    /// are provided for adding a new pair as well as for setting
-    /// a key to a single value. All keys are strings but values
-    /// may be of any type. Null values are not permitted, since
-    /// a null entry represents the absence of the key.
-    /// </summary>
-    public class PropertyBag : IPropertyBag
-    {
-        private readonly Dictionary<string, IList> inner = new Dictionary<string, IList>();
-
-        #region IPropertyBagMembers
-
-        /// <summary>
-        /// Adds a key/value pair to the property set
-        /// </summary>
-        /// <param name="key">The key</param>
-        /// <param name="value">The value</param>
-        public void Add(string key, object value)
-        {
-            Guard.ArgumentNotNull(value, "value");
-
+// ***********************************************************************

```

Comment:
Is this a change in line endings?

Reply:
Ah... yes. Oops. Can I push to split this into two commits?

Reply:
TestCaseAttribute has the same problem.

Reply:
@rprouse I now have an initial commit converting these two files to CRLF, but I'm wondering if that was the wrong thing to do. Shouldn't Git be autoconverting for all files unless specified otherwise in .gitattributes?